src/plugins/cableCar.ts

```diff
@@ -43,6 +43,7 @@ export class CableCar {
     document.addEventListener('DOMContentLoaded', this.boundScanner)
     document.addEventListener('turbolinks:load', this.boundScanner)
     document.addEventListener('turbo:load', this.boundScanner)
+    document.addEventListener(AJAX_EVENTS.ajaxComplete, this.boundPerform)
```

Comment:
Good call. No reason to have these element level when they all bubble up to the document